packages/registry/README.md

```diff
@@ -0,0 +1,7 @@
+### Registry
```

Comment:
Can probably wait, but I really how you called it "Modules Store" in your PR, maybe we should call it that? What do you think? Probably need input from more team members

Reply:
I also though about calling it something like "Modules state". "Store" might be confusing, people might think about physical storage of the modules (artifacts / js bundles).

Reply:
True that, let's postpone the decision for now then

Reply:
Will move it to issue for further discussion.